README.md

```diff
@@ -174,6 +174,28 @@ for our complete browser support matrix:
 The polyfills may work in older browsers, however require additional polyfills (such as classList, or other [platform](https://github.com/webcomponents/webcomponents-platform)
 polyfills) to be used. We cannot guarantee support for browsers outside of our compatibility matrix.
 
+## Known Issues
+
+  * [Style encapsulation (inline styling in components) does not work out of the box](#shadycss)
+  * [Custom element's constructor property is unreliable](#constructor)
+  * [ShadyCSS: :host(.zot:not(.bar:nth-child(2))) doesn't work](#nestedparens)
+
+### Style encapsulation (inline styling in components) does not work out of the box. <a id="shadycss"></a>
+The ShadowDOM polyfill does not properly support CSS in ShadowDoM out of the box:
+ * Any styles inside components have an effect on the whole document (instead of on the component only - the encapsulation is broken).
+ * Any shadow-dom specific selectors (like `:host`) do not work.
+ 
+You can fix those issues by manually calling the `ShadyCSS` APIs. See [ShadyCSS usage](https://github.com/webcomponents/shadycss#usage).
```

Comment:
I still think this section is misleading, as calling the `ShadyCSS` APIs is intended usage behavior when using this polyfill.
What about the following?

```md
To achieve Shadow DOM behavior with regards to CSS, you need to manually call `ShadyCSS` APIs (See [ShadyCSS usage](https://github.com/webcomponents/shadycss#usage) for more details).
If you do not call ShadyCSS, style definitions will have effect on the whole document.
This includes style definitions inside custom elements.
Moreover, `ShadyCSS` will also rewrite (and thus make them work) Shadow DOM-specific selectors like `:host`.
```

Reply:
I understand my wording is incorrect. However the issue that users like me encounter is that , contrary to other polyfills, including ShadowDOM is not enough to make it work according to the web components standard.
And I am trying to make clear what the symptoms of not calling the `ShadyCSS` api are.
Does that seem to be a proper goal ?

Reply:
Yes absolutely agreed that is the goal. My proposed edit is intended to reach that goal as well, is it not?

Reply:
Maybe we should just include a full example in the README here, including how to use `ShadyCSS`, rather than linking to a different doc. Then we can also add a section "CSS selectors not working" and add the solution there. WDYT?

Reply:
That would definitely help. However, I do not volunteer for it as I feel I do not know enough to come up with a meaningful example. Would be happy to review though.

Reply:
I am quite busy atm. I hope to get to this some time next week.